rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -212,15 +256,15 @@ def _calculate_staking_balances(
 
     @staticmethod
     def _get_staking_history(
-        staking_balances: Dict[ChecksumAddress, List[ADXStakingBalance]],
-        staking_events: ADXStakingEvents,
-        tom_pool_incentive: Optional[TomPoolIncentive] = None,
+            staking_balances: Dict[ChecksumAddress, List[ADXStakingBalance]],
+            staking_events: ADXStakingEvents,
+            tom_pool_incentive: TomPoolIncentive,
```

Comment:
Another thing that is needed for PnL is history in a given period. So if you can make this work for specific from/to timestamps this would really help.